frontend/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -31,18 +33,30 @@ class ErrorBoundary extends React.Component<Props, State> {
   }
 
   render() {
-    const { hasError, error, errorInfo } = this.state;
+    const { hasError, error, errorInfo, modalOpen } = this.state;
     const { children } = this.props;
 
-    const handleClick = () => {
-      localStorage.clear();
+    const gotoDegrees = () => {
       window.location.href = '/degree-wizard';
```

Comment:
nitpick: is it possible/could we use `useNavigate` for the redirection?

Reply:
I tried, but since the error boundary is outside the router, it is not possible i dont think?